package.json

```diff
@@ -5,6 +5,7 @@
     "@hashicorp/flight-icons": "^2.5.0",
     "@hashicorp/mktg-global-styles": "^4.2.0",
     "@hashicorp/mktg-logos": "^1.3.2",
+    "@hashicorp/react-product-badge": "^0.1.1",
```

Comment:
I don't think we want this dependency installed at the root. You can fix this by running the following in the root directory:

```
npm uninstall @hashicorp/react-product-badge
npm install @hashicorp/react-product-badge --workspace=@hashicorp/react-card
```

That will ensure that consumers who install `@hashicorp/react-card` get the `@hashicorp/react-product-badge` dependency.

Reply:
Gah, my bad! Will do!

Reply:
Commit [here](https://github.com/hashicorp/react-components/pull/748/commits/0d5334ef7c6a10c3f1cc292b992dd1e8f64b2b56).